src/app/modules/project/edit/edit.component.html

```diff
@@ -55,33 +55,32 @@ <h3>Short description*</h3>
         <div class="col-12">
           <div>
             <h3>Description*</h3>
-            <textarea formControlName="description" required [(ngModel)]="project.description"></textarea>
+            <textarea formControlName="description" required [ngModel]="project?.description"></textarea>
           </div>
         </div>
       </div>
     </form>
 
 
     <form [formGroup]="editContributorForm" class="project-form">
-      <p><b>Add project members</b></p>
+      <p><b>Edit project members</b></p>
       <div class="row">
         <div class="col-12">
-          <div *ngFor="let collaborator of project.collaborators">
-            <div class="row">
-              <div class="col-5">
-                <h3>Contributor full name</h3>
-                <input formControlName="fullName" required [(ngModel)]="collaborator.fullName">
-              </div>
-              <div class="col-5">
-                <h3>Contributor function</h3>
-                <input formControlName="role" required [(ngModel)]="collaborator.role">
-              </div>
-              <div class="col-2 add-contributor-col">
-                <button type="button" class="btn btn-icon btn-primary add-contributor-button" aria-label="settings"
-                  (click)="onClickAddContributor()" [disabled]="!editContributorForm.valid">
-                  +
-                </button>
-              </div>
+          <div class="row">
+            <div class="col-5">
+              <h3>Collaborator full name</h3>
+              <input formControlName="fullName" required>
+            </div>
+            <div class="col-5">
+              <h3>Collaborator function</h3>
```

Comment:
I think we should also display 'role' as the name for this field instead of function

Reply:
Nice one, will change.